### ISSUE TITLE
Build a User Interface

### ISSUE THREAD

**Issue by swaaz** [swaaz/Bulk-Mailer#10]
Build a minimal user interface using HTML, CSS and JS or any other framework

**Comment by jasonappah** [comment]
I can work on this. Just to be sure, this is intended to be run locally on a user's computer and not on a server, right?

**Comment by bernandez** [comment]
Can a framework like Django be used?

**Comment by bernandez** [comment]
Can Django be useful?

**Comment by swaaz** [comment]
Yes, @jasonappah you are right. But I planned to make it run on the server but I don't think we can use GitHub pages for that. Do you have any suggestions? 
Feel free to create your pull request to `hacktoberfest` branch **not** to `master` branch directly

**Comment by swaaz** [comment]
@bernandez you can use any framework.
Feel free to create your pull request to `hacktoberfest` branch **not** to `master` branch directly

**Comment by jasonappah** [comment]
If you want it to run on a server then we should definitely plan on using something like Flask or Django. I think Django will probably be easier though

**Comment by swaaz** [comment]
@jasonappah yea Django is also fine. But I feel flask would be easier and convenient.

**Comment by jasonappah** [comment]
Cool - I guess first steps should be breaking main.py up into separate functions and getting a basic API going. Also, since we're going to have to run a webserver anyway, wouldn't it be a good idea to use oauth to get user creds instead of storing them in plain text and do the emailing through Gmail's API?

**Comment by bernandez** [comment]
And also please this my firs time attending Hacktoberfest and also working on Git so am kinda new to all this

**Comment by swaaz** [comment]
@jasonappah @bernandez you can divide work yourselves. Feel free to tag me in this issue anytime when you guys have doubt. Happy to see you working together.
